tsc/meetings/2019-03-21.md

```diff
@@ -0,0 +1,108 @@
+Minutes from 11th OpenVDB TSC meeting, Mar. 14, 2019
+
+Attendees: *Nick* A., *Dan* B. (late), *Peter* C., *Jeff* L.
+
+Additional Attendees: Bruce Chernia (Intel), Andrew Pearce (DWA),
+Thanh Ha (LF)
+
+Agenda:
+
+1) Confirmation of quorum
+2) Selection of secretary
+3) Update on CMake
+4) Reno Change
+5) Documentation.
+6) Warnings as Errors
+7) Unifying SOP Labels (postponed)
+8) JIRA is not working
+9) Deprecation (added)
+10) Next Meeting
+
+1) A quorum was confirmed.  Nick was selected to chair the meeting.
+
+2) Secretary - Jeff Lait
+
+3) Update on CMake from Nick
+
+3a) CMake
+
+The PR is in a good place.  Required more work than expected, but mostly
+tidying up and testing on Mac and Linux.   Not tested on Maya-Windows.
+
+Blosc, ILM-Half, TBB, and a few others do not have CMake modules yet.
+
+Houdini and GLFW modules done - using the CMake examples from SideFX.
+
+There will be a PR add a custom module for numpy to avoid it requiring CMake
+3.14.
+
+Call for everyone to test the CMake on their own platforms.
+
+3b) Circle CI and Travis
+
+The TAC has enabled CircleCI so we can move forward with that and it should
+work.  Dan will help with CircleCI additions to the CMake PR so it can
+be merged with very basic CI support (Houdini 17 / ABI 6)
```

Comment:
Plan is to initially switch on CircleCI as an optional check and leave TravisCI as required check while we build up CircleCI support.